webapp/components/_new/features/MySomethingList/MySomethingList.vue

```diff
@@ -56,7 +56,7 @@
         >
           {{ isEditing ? $t('actions.save') : texts.addButton }}
         </base-button>
-        <base-button v-if="isEditing" id="cancel" danger @click="handleCancel()">
+        <base-button v-if="isEditing" id="cancel" @click="handleCancel()">
```

Comment:
This
```suggestion
        <base-button 
          v-if="isEditing"
          id="cancel"
          icon="close"
          @click="handleCancel()"
        >
```
and on line 55 it would be nice to have a ``icon="check"``